resources/views/layouts/_twitter_embeds.blade.php

```diff
@@ -0,0 +1,30 @@
+@if ($appId = config('services.twitter.embeds.enabled'))
+<script>
+    (function ($, block) {
+        function embedTwitterWidgetScript() {
+            var notEmbedded = $('script[src="//platform.twitter.com/widgets.js"]').length === 0;
+            if (notEmbedded) {
+                $('body').append('<script async src="//platform.twitter.com/widgets.js" charset="utf-8"><\/script>');
+            }
+        }
+
+        function changeMatchForEmbed(link, content) {
+            content.innerHTML = content.innerHTML.replace(
+                link,
+                '<blockquote class="twitter-tweet" data-lang="en"><a href="'+link+'">'+link+'</a></blockquote>'
+            )
+        }
+
+        function regexForMatch(id, content) {
+            var regex = /((http(s|):\/\/)|)twitter.com\/\w*\/status\/\d*/g;
+            var match = regex.exec($(content).text())
+            if (match && match[0]) {
+                changeMatchForEmbed(match[0], content)
+                embedTwitterWidgetScript();
+            }
+        }
+
+        $(block).each(regexForMatch)
+    })($, '.forum-content > p');
+</script>
```

Comment:
Hey Dries, I made the changes you requested except for the Vue component, which I can still do but I wanted to see if you liked this jquery implementation better, the only reason I hesitate on Vue component is to me Vue seems better suited for pulling pieces out throughout the entire div#app instead of single blocks of content which single blocks of content replacement seems more in line with what a jquery plugin would do.